database/import.go

```diff
@@ -146,75 +90,39 @@ func getHeadersFile(preparedDbFilePath string, log *zerolog.Logger) (*os.File, s
 	return tmpHeadersFile, tmpHeadersFilePath, nil
 }
 
-func importHeadersFromFile(repo repository.Headers, inputFile *os.File, log *zerolog.Logger) (int, error) {
-	log.Info().Msg("Inserting headers from file to the database")
-
-	// Read from the beginning of the file
-	if _, err := inputFile.Seek(0, 0); err != nil {
-		return 0, err
-	}
-
-	reader := csv.NewReader(inputFile)
-	_, err := reader.Read() // Skipping the column headers line
-	if err != nil {
-		return 0, err
-	}
-
-	previousBlockHash := chainhash.Hash{}
-	rowIndex := 0
-
-	for {
-		batch := make([]domains.BlockHeader, 0, insertBatchSize)
+func dropHeadersFile(tmpHeadersFile *os.File, tmpHeadersFilePath string, log *zerolog.Logger) {
+	_ = tmpHeadersFile.Close()
```

Comment:
Are you sure, the error from `Close` can be skipped?

Reply:
Even if the file has already been closed, we would like to remove it. Please note that we do not want to stop the application even if errors occur here.

Reply:
Got it 👍